index.html

```diff
@@ -0,0 +1,147 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+  <meta charset="UTF-8" />
+  <link rel="icon" type="image/svg+xml" href="favicon.svg" />
+  <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+  <title>Tech Academy</title>
+  <link rel="stylesheet" href="https://use.typekit.net/kjf8squ.css">
+  <script src="https://kit.fontawesome.com/67c66657c7.js"></script>
+  
+  <link rel="stylesheet" href="style.css">
+</head>
+
+<body>
+  <header class="primary-header">
+    <div class="container">
+          <nav class="navigation">
+            <div class="menu-button"><img id="humberger" src="img/menu.svg" alt="menu-button"></div>
+            <a href="index.html"><img class="logo"src="img/logo-2.png" alt="Manage" /></a>
+          <div class="nav-bar">
```

Comment:
![image](https://user-images.githubusercontent.com/93039359/196925379-6df8889e-c57a-4801-80b1-e4020b902971.png)

### CindyShin Design
![image](https://user-images.githubusercontent.com/93039359/196925577-91ac05d8-fff7-4fec-8d5b-3a8a55be6bba.png)


- [ ] Your primary-header should not have a background. It should come on top the already existing background as you can see in the CindyShin Design. Kindly make your primary-header background less :+1: to follow the design :blush: